ReadMe.rst

```diff
@@ -65,6 +65,24 @@ append the root path to the build variable string.*
 After this the package should be discoverable to can be used in the build like
 any other dependency.
 
+SetupDataPkg's Tools
+====================
+The SetupDataPkg/Tools folder contains Utilities that can enable working with Setup data
+
+========================================== ===========================================================================================
+ Utility                                   Usage
+========================================== ===========================================================================================
+SetupDataPkg\Tools\ConfigEditor.py         Configuration utility for creating/modifying SVD,XML configurations
+SetupDataPkg\Tools\GenCfgData.py           Utility to generate output configuration structres from input YAML configurations
+SetupDataPkg\Tools\GenNCCfgData_test.py    Utility to generate output configuration fron input XML configuration
+SetupDataPkg\Tools\VariableList.py         Utility to generate EFI dmpstore foramtted binaries, or csv file based on XML configuration
+SetupDataPkg\Tools\WrapAndSetSvd.py        Utility to generate Dfci formatted output from input SVD file.

```

Comment:
Double slash for the paths

Reply:
switched to / as per https://github.com/microsoft/mu_feature_config/pull/79#discussion_r1048954684